Makefile

```diff
@@ -1,4 +1,4 @@
-include .env
+-include .env
```

Comment:
Should not exit for missing `.env` file. Not everyone will be running integration tests